modules/tts/tts-adapters/tts-adapter-sapinative/src/test/java/ignore

```diff
@@ -1 +1,7 @@
-org/daisy/pipeline/tts/sapinative/impl/SAPINativeTest.java
+# Tests are not run by default. It only makes sense to run them on Windows.
+# Don't enable OnecoreTest and SAPITest when SAPIServiceTest is
+# also enabled. Due to concurrent attempt to load or dispose of
+# library, we cannot run them concurrently.
```

Comment:
Can you elaborate a bit on this? What exactly is the problem? Is there are way to overcome it?

Reply:
I don't remember exactly but if i recall, when launching the test simultaneously, it would try to load the dll multiple times and it would make the tests failed.

I'll need to retest that to exactly know what was going on on the jvm side, if it was a runtime exeception due to dll reloading or if it was something linked to the libs initialization process that would crash the rest of the code (possibly due to reallocation of global variables requested in the tests initialization).

Reply:
Or was it an issue due to parallel execution of tests? (because you mention "concurrent")

Reply:
Fixed.